cli/resources/META-INF/native-image/clojure-lsp/clojure-lsp/reflect-config.json

```diff
@@ -17,49 +17,6 @@
   "name":"clojure.lang.Var",
   "allPublicMethods":true
 },
```

Comment:
We certainly need to re-check that, we had many changes on deps since started using graal so there is a chance that we don't need more reflection configs in this file. Let's do in another PR though